CardanoSharp.Wallet/Models/Transactions/AuxiliaryData.cs

```diff
@@ -2,23 +2,23 @@
 
 namespace CardanoSharp.Wallet.Models.Transactions
 {
```

Comment:
Looks like my editor auto spaced the tabbing here. I can revert this if you want